app/templates/organen/orgaan.hbs

```diff
@@ -20,15 +20,20 @@
   </Group>
 </AuToolbar>
 
-<Form::Instance
-  @instanceId={{this.model.instanceId}}
-  @form={{this.model.form}}
-  @isEditable={{false}}
-/>
+<div class="au-o-box">
+  <Form::Instance
+    @instanceId={{this.model.instanceId}}
+    @form={{this.model.form}}
+    @isEditable={{false}}
+  />
+
+  <div class="au-o-layout">
+    <AuButton
+      class="au-u-margin-bottom"
+      @skin="secondary"
+      {{on "click" this.edit}}
+    >Pas aan</AuButton>
 
-<AuToolbar @border="top" @size="large" as |Group|>
```

Comment:
Why did you remove the toolbar here, seems like a good practice to use those?

Reply:
Because we wanted to make the "Pas aan" look inline with the form. It might be strange to have it in the toolbar as well no?

Reply:
And karel wanted the todo to be directly under the form